BuyBitcoin/settings.py

```diff
@@ -104,8 +112,7 @@
         'PORT':     '',
     }
 else:
-    DATABASES['default'] = dj_database_url.config(
-        default="postgres://bitcoin:stupid_pw@localhost/buy_bitcoin")
+    DATABASES['default'].update(dj_database_url.config(conn_max_age=500))
```

Comment:
This line will overwrite line 100, no?

Reply:
Nope, this only selects if it finds the env